.goreleaser.yml

```diff
@@ -46,3 +45,66 @@ dockers:
   - "--label=org.opencontainers.image.version={{.Version}}"
   - "--label=org.opencontainers.image.source={{.GitURL}}"
   - "--platform=linux/amd64"
+- image_templates:
+  - 'skibish/ddns:{{ .Tag }}-arm64'
+  dockerfile: Dockerfile
+  use: buildx
+  build_flag_templates:
+  - "--pull"
+  - "--label=org.opencontainers.image.created={{.Date}}"
+  - "--label=org.opencontainers.image.name={{.ProjectName}}"
+  - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+  - "--label=org.opencontainers.image.version={{.Version}}"
+  - "--label=org.opencontainers.image.source={{.GitURL}}"
+  - "--platform=linux/arm64"
+- image_templates:
+  - 'skibish/ddns:{{ .Tag }}-armv6'
+  dockerfile: Dockerfile
+  use: buildx
+  build_flag_templates:
+  - "--pull"
+  - "--label=org.opencontainers.image.created={{.Date}}"
+  - "--label=org.opencontainers.image.name={{.ProjectName}}"
+  - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+  - "--label=org.opencontainers.image.version={{.Version}}"
+  - "--label=org.opencontainers.image.source={{.GitURL}}"
+  - "--platform=linux/arm/v6"
+- image_templates:
+  - 'skibish/ddns:{{ .Tag }}-armv7'
+  dockerfile: Dockerfile
+  use: buildx
+  build_flag_templates:
+  - "--pull"
+  - "--label=org.opencontainers.image.created={{.Date}}"
+  - "--label=org.opencontainers.image.name={{.ProjectName}}"
+  - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+  - "--label=org.opencontainers.image.version={{.Version}}"
+  - "--label=org.opencontainers.image.source={{.GitURL}}"
+  - "--platform=linux/arm/v7"
+- image_templates:
+  - 'skibish/ddns:{{ .Tag }}-i386'
+  dockerfile: Dockerfile
+  use: buildx
+  build_flag_templates:
+  - "--pull"
+  - "--label=org.opencontainers.image.created={{.Date}}"
+  - "--label=org.opencontainers.image.name={{.ProjectName}}"
+  - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+  - "--label=org.opencontainers.image.version={{.Version}}"
+  - "--label=org.opencontainers.image.source={{.GitURL}}"
+  - "--platform=linux/386"
+docker_manifests:
+- name_template: skibish/ddns:{{ .Tag }}
+  image_templates:
+  - skibish/ddns:{{ .Tag }}-amd64
+  - skibish/ddns:{{ .Tag }}-arm64
+  - skibish/ddns:{{ .Tag }}-armv6
+  - skibish/ddns:{{ .Tag }}-armv7
+  - skibish/ddns:{{ .Tag }}-i386
+- name_template: skibish/ddns:latest
+  image_templates:
+  - skibish/ddns:{{ .Tag }}-amd64
```

Comment:
because these manifests are for `latest` images, let's change the tags from `{{.Tag}}-<os>` to `latest-<os>`